packages/scan/src/Scan.spec.ts

```diff
@@ -304,6 +304,59 @@ describe('Scan', () => {
     });
   });
 
+  describe('expect with failFast: false', () => {
+    it('should wait for scan completion', async () => {
+      when(mockedScans.getScan(id)).thenResolve({
+        status: ScanStatus.DONE
+      });
+
+      await scan.expect(Severity.LOW, { failFast: false });
+
+      verify(mockedScans.getScan(id)).once();
+    });
+
+    it('should wait for scan completion after a few iterations', async () => {
+      when(mockedScans.getScan(id))
+        .thenResolve({
+          status: ScanStatus.RUNNING,
+          issuesBySeverity: [{ number: 1, type: Severity.LOW }]
+        })
+        .thenResolve({
+          status: ScanStatus.RUNNING
```

Comment:
The issue could be not disappear 😄
```suggestion
          status: ScanStatus.RUNNING,
          issuesBySeverity: [{ number: 1, type: Severity.LOW }]

```

Reply:
fixed